src/components/SchedulePage/Schedule.js

```diff
@@ -37,6 +37,7 @@ const SchedulePage = () => {
 		<Typography align='left' component='h1' variant='h4' className={classes.title}>
 			Schedule
 		</Typography>
+		<p style={{ fontSize: '24px', paddingBottom: '50px' }}>Coming soon...</p>
```

Comment:
For fontsize, you should use `theme.typography.fontSize`. And for stuff like padding and margin, you should use `theme.spacing()`. This makes our code much more maintainable and uniform.